examples/apptainer-rootful.yaml

```diff
@@ -4,12 +4,12 @@
 
 images:
 # Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
-- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20240423/ubuntu-24.04-server-cloudimg-amd64.img"
+- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20240710/ubuntu-24.04-server-cloudimg-amd64.img"
```

Comment:
Now https://cloud-images.ubuntu.com/releases/24.04/release-20240725/ is available